backend/request.ts

```diff
@@ -409,17 +422,23 @@ export async function parseFilterUsersRequest(
     let mail = undefined;
     let isCoachFilter = undefined;
     if ("isCoachFilter" in req.body) {
-        isCoachFilter = req.body.isCoachFilter === "true";
+        isCoachFilter = req.body.isCoachFilter.toString() === "true";
     }
     let isAdminFilter = undefined;
     if ("isAdminFilter" in req.body) {
-        isAdminFilter = req.body.isAdminFilter === "true";
+        isAdminFilter = req.body.isAdminFilter.toString() === "true";
     }
     if (
-        "statusFilter" in req.body &&
-        req.body.statusFilter !== "ACTIVATED" &&
-        req.body.statusFilter !== "PENDING" &&
-        req.body.statusFilter !== "DISABLED"
+        ("statusFilter" in req.body &&
+            req.body.statusFilter !== "ACTIVATED" &&
+            req.body.statusFilter !== "PENDING" &&
+            req.body.statusFilter !== "DISABLED") ||
+        ("isCoachFilter" in req.body &&
+            req.body.isCoachFilter.toString() !== "true" &&
+            req.body.isCoachFilter.toString() !== "false") ||
+        ("isAdminFilter" in req.body &&
+            req.body.isAdminFilter.toString() !== "true" &&
+            req.body.isAdminFilter.toString() !== "false")
```

Comment:
we could define our own kind of StringBoolean to check if this is valid

Reply:
huh, is the `isCoachFilter` not a boolean?

Reply:
There currently is no way to set these filters to false in the frontend. So we might not even need to check if they are true or false.